examples/demo.js

```diff
@@ -11,6 +11,24 @@ document.addEventListener('DOMContentLoaded', function () {
         }
     });
 
+    /* language section */ 
+    var request = new XMLHttpRequest()
+
+    request.open('GET', 'https://api.github.com/users/publiclab/repos', true);
+
+    request.onload = function () {
+    var data = JSON.parse(this.response);
+    var statusHTML = '';
+    $.each(data, function (i, status) {
+        statusHTML += '<div class="conatiner-language">';
+        statusHTML += '<div class="repo-name"><a href="' + status.html_url + '">' + status.name + '</a></div>';
+        statusHTML += '<divs class="lang-name">' + status.language + '</div>';
```

Comment:
This is looking very good - however, could we use standard Bootstrap badges for these languages?  https://getbootstrap.com/docs/4.4/components/badge/

Also, i think `divs` is a typo!

Thanks for your work on this! If there's a way to ensure languages are a little closer associated with the project names, that's probably good too, as currently it seems like maybe it's easy to confuse which language they correspond to. What if we did it like:

```
Project Name
[language]
```